src/components/TruckSizeSelect/TruckSizeSelect.css

```diff
@@ -19,8 +19,10 @@
 
 .truck-size-container {
     display: flex;
+    max-width: 600px;

```

Comment:
Why this? I removed it so the selector is centered. But my design feel might be off here :)

Reply:
The buttons get widely separated in full screen ^^ Width should be limited somewhere, but probably better done in a higher up wrapper around the inputs.

Reply:
Same with the sliders that get uncomfortable when too wide

Reply:
Oh I see yeah use 
```
<div className="container">
..stuff
</div>
```

That should be good.